### DISCUSSION TITLE
How to enable "fast-diff"  rbd image with csi?

### DISCUSSION THREAD

**Discussion by zhucan** [ceph/ceph-csi#3300] [Q&A]
warning: fast-diff map is not enabled for csi-vol-be92926f-1893-11ed-a79b-de179ebd0891. operation may be slow.

**Comment by Madhu-1** [comment]
You can set it in rbd storageclass https://github.com/ceph/ceph-csi/blob/54fe4158a0355d1e2f3f63f0e67474fab0d0d405/examples/rbd/storageclass.yaml#L37

**Comment by zhucan** [comment]
Thanks @Madhu-1
